icepack/models/shallow_ice.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (C) 2020 by Jessica Badgeley <badgeley@uw.edu>
+#
+# This file is part of icepack.
+#
+# icepack is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# The full text of the license can be found in the file LICENSE in the
+# icepack source directory or at <http://www.gnu.org/licenses/>.
+
+import firedrake
+from firedrake import inner, grad, dx, ds
```

Comment:
There's no boundary forcing, so you could leave off importing `ds`; likewise the water density isn't used here either.